Source/player.h

```diff
@@ -33,6 +33,7 @@ namespace devilution {
 #define MAX_SPELL_LEVEL 15
 #define PLR_NAME_LEN 32
 
+static constexpr uint8_t NUM_HOTKEYS = 16;

```

Comment:
[readability-identifier-naming](https://clang.llvm.org/extra/clang-tidy/checks/readability-identifier-naming.html), devilutionX uses `CamelCase` for constexpr variables.

The type is used in comparison with size_t variables and is used to define the size of arrays, `size_t` is more appropriate than `uint8_t`
```suggestion
static constexpr size_t NumHotkeys = 16;
```